app/selectors/favorites.js

```diff
@@ -0,0 +1,4 @@
+export function getFavoriteTrack(state, artist, title) {
+  return state.favorites.tracks
+    .find(track => track.artist.name === artist && track.name === title);
```

Comment:
I'm considering simply using `toLowerCase()` here or more advanced comparison with 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare

In the second option, the main question is how we should treat the accents. `'a' = 'á'`?

Reply:
Yep, use `_.deburr`

Reply:
Done